smartcar/demo
# Validate client id and secret as UUIDs before running the demo

## Test Plan
- Run the demo locally with invalid client ID and invalid secret. Make sure errors are thrown.
- Run the demo locally with valid client ID and valid secret. Make sure demo runs successfully.
